### PR TITLE
Release 1.3 changes for PowerMax 

### PR DESCRIPTION
**Pull request by shanmydell**
# Description
Release 1.3 changes for PowerMax 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/243 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by mahesr2 — DISMISSED** [review]

File: content/docs/csidriver/installation/helm/powermax.md [hunk 1]
```diff
@@ -181,14 +181,17 @@ CRDs should be configured during replication prepare stage with repctl as descri
...
-| serviceLevel  | This parameter must mention one of the Service Levels on the PowerMax array. This value is used to create the default storage class.   | Yes| "Bronze"     |
 | skipCertificateValidation | Skip client-side TLS verification of Unisphere certificates | No | "True" |
 | transportProtocol  | Set the preferred transport protocol for the Kubernetes cluster which helps the driver choose between FC and iSCSI when a node has both FC and iSCSI connectivity to a PowerMax array.| No | Empty|
 | nodeNameTemplate | Used to specify a template that will be used by the driver to create Host/IG names on the PowerMax array. To use the default naming convention, leave this value empty.  | No | Empty|
 | fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType` | No | "ReadWriteOnceWithFSType" |
+| version | Current version of the driver. Don't modify this value as this value will be used by the install script. | Yes | v2.3.0 | 
+| images | Defines the container image, used for the driver container.  | Yes | driverRepository: dellemc |
```
Comment (boyamurthy) [inline content/docs/csidriver/installation/helm/powermax.md:194]: add images as section and under that add "driverRepository" as parameter with default value "dellemc" and it is required field.

File: content/docs/csidriver/installation/helm/powermax.md [hunk 2]
```diff
@@ -232,6 +235,11 @@ CRDs should be configured during replication prepare stage with repctl as descri
 | sidecarProxyImage | Image for csm-authorization-sidecar. | No | " " |
 | proxyHost | Hostname of the csm-authorization server. | No | Empty |
 | skipCertificateValidation | A boolean that enables/disables certificate validation of the csm-authorization server. | No | true |
+| **replication** | [Replication](../../../../replication/deployment) is an optional feature to enable replication & disaster recovery capabilities of PowerMax to Kubernetes clusters.| - | - |
+| enabled                  | A boolean that enables/disables replication feature. |  No      |   false   |
+| image | Image for dell-csi-replicator sidecar. | No | " " |
```
Comment (boyamurthy) [inline content/docs/csidriver/installation/helm/powermax.md:240]: pls add default image name .

File: content/docs/csidriver/installation/helm/powermax.md [hunk 3]
```diff
@@ -181,14 +181,17 @@ CRDs should be configured during replication prepare stage with repctl as descri
 | kubeletConfigDir | Specify kubelet config dir path | Yes | /var/lib/kubelet |
 | imagePullPolicy |  The default pull policy is IfNotPresent which causes the Kubelet to skip pulling an image if it already exists. | Yes | IfNotPresent |
 | clusterPrefix | Prefix that is used during the creation of various masking-related entities (Storage Groups, Masking Views, Hosts, and Volume Identifiers) on the array. The value that you specify here must be unique. Ensure that no other CSI PowerMax driver is managing the same arrays that are configured with the same prefix. The maximum length for this prefix is three characters. | Yes  | "ABC" |
+| logLevel | CSI driver log level. Allowed values: "error", "warn"/"warning", "info", "debug". | Yes | "debug" |
+| logFormat | CSI driver log format. Allowed values: "TEXT" or "JSON". | Yes | "TEXT" |
```
Comment (boyamurthy) [inline content/docs/csidriver/installation/helm/powermax.md:185]: please add them under global section .

**Review by boyamurthy — COMMENTED** [review]
the storageresouce and service level from the parameters table .

File: content/docs/csidriver/installation/helm/powermax.md (hunk 2 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/powermax.md:240]: Default image will be set to none , when this value is not provided

File: content/docs/csidriver/installation/helm/powermax.md (hunk 3 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/powermax.md:185]: Addressed

File: content/docs/csidriver/installation/helm/powermax.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/powermax.md:194]: addressed

**Review by boyamurthy — APPROVED** [review]

**Review by mahesr2 — DISMISSED** [review]
LGTM

**Review by mahesr2 — APPROVED** [review]

**Pull request merged**